Flow.Launcher/ViewModel/SettingWindowViewModel.cs

```diff
@@ -555,6 +555,39 @@ public FamilyTypeface SelectedResultFontFaces
         public string Github => Constant.GitHub;
         public static string Version => Constant.Version;
         public string ActivatedTimes => string.Format(_translater.GetTranslation("about_activate_times"), Settings.ActivateTimes);
+        public string CheckLogFolder
+        {
+            get 
+            { 
+                DirectoryInfo dirInfo = new DirectoryInfo(Path.Combine(DataLocation.DataDirectory(), Constant.Logs, Constant.Version));
+                long size = 0;
+                foreach (FileInfo fi in dirInfo.GetFiles("*", SearchOption.AllDirectories))
+                {
+                    size += fi.Length;
+                }
+            return string.Format(_translater.GetTranslation("clearlogfolder")) + " (" + FormatBytes(size) + ")" ;
```

Comment:
https://stackoverflow.com/a/22111211

Try this instead? (it looks more elegant XD)

Reply:
Also please use string interpolation instead of string.format